File: Python/codeup_1092_함께 푸는 날.py
a, b, c = input().split()

a = int(a)
b = int(b)
c = int(c)

day = 1
i = 0
while i < 1:
    if(day % a == 0 and day % b == 0 and day % c == 0):
        break
    day += 1
    
print(day)

File: C++/codeup_1092_함께 푸는 날.cpp
#include<iostream>

using namespace std;

int main()
{

    int a, b, c;
    int day = 1;

    cin >> a >> b >> c;

    for(int i = 0; ; i++)
    {
        if(day % a == 0 && day % b == 0)
            if(day % c == 0)
                break;

        day++;
    }

    cout << day << endl;

   return 0;
}

